assets/dev/js/frontend/handlers/image-carousel.js

```diff
@@ -176,4 +176,38 @@ export default class ImageCarousel extends elementorModules.frontend.handlers.Sw
 			this.swiper.slideToLoop( this.getEditSettings( 'activeItemIndex' ) - 1 );
 		}
 	}
+
+	getSpaceBetween( device = null ) {
+		return elementorFrontend.utils.controls.getResponsiveControlValue( this.getElementSettings(), 'image_spacing_custom', 'size', device ) || 0;
+	}
+
+	updateSpaceBetween( propertyName ) {
+		const deviceMatch = propertyName.match( 'image_spacing_custom_(.*)' ),
+			device = deviceMatch ? deviceMatch[ 1 ] : 'desktop',
+			newSpaceBetween = this.getSpaceBetween( device );
+
+		if ( 'desktop' !== device ) {
+			this.swiper.params.breakpoints[ this.getDeviceBreakpointValue( device ) ].spaceBetween = newSpaceBetween;
```

Comment:
Why not get the value directly from the `elementorFrontend.config.responsive.activeBreakpoints` array?
Why do you need the dictionary?

Reply:
@udidol Done.